proxy/envoy/resources.go

```diff
@@ -419,6 +427,7 @@ type Cluster struct {
 	MaxRequestsPerConnection int               `json:"max_requests_per_connection,omitempty"`
 	Hosts                    []Host            `json:"hosts,omitempty"`
 	Features                 string            `json:"features,omitempty"`
+	SSLContext               *SSLContext       `json:"ssl_context,omitempty"`
```

Comment:
Let's remove this since it's not used. The schema is slightly different anyways for cluster SSL context.